docs/subsystems/icons.md

```diff
@@ -30,28 +30,186 @@ using icons. Typically, this means:
 
 ## Adding a new icon
 
-A new feature may require a new icon to represent it, for example to be used
-next to a menu option. The issue you're working on may not have an icon
-specified upfront. In this case, you should:
+A new feature, such as a new menu option, may require a new icon to represent
+it. The issue you're working on may not have an icon specified upfront. In that
+case, you should:
 
 1. Prototype using a [Lucide icon](https://lucide.dev/icons/), which is the
-   default source for icons in Zulip. SVG files must be added under
-   `web/shared/icons/` (don't forget to `git add`) to be used.
+   preferred source for icons in Zulip. New SVG files must be placed in the
+   `web/shared/icons/` directory to be used (don't forget to `git add` new
+   icons along with your other changes).
 
 1. **When your feature is nearing completion**, post in the [appropriate
    channel](https://zulip.com/development-community/#where-do-i-send-my-message)
    in the Zulip development community to discuss what icon should be used (e.g.,
    #design for web app icons). You can use the discussion thread linked from the
    issue you're working on if there is one.
 
-1. Once there is general consensus, @-mention Zulip's designer (Vlad Korobov) to
-   ask him to prepare the icon to be used.
+1. Once there is general consensus on the icon to be used, be sure to
+   [prepare the icon's SVG file](#preparing-icons-for-use-with-zulip)
+   so that it displays correctly in Zulip.
 
 1. Follow the [attribution guidelines](../contributing/licensing.md)
-   to document the icon source.
+   to document the icon's source in `docs/THIRDPARTY/`.
 
-## Changing an icon
+## Preparing icons for use with Zulip
+
+You may discover that your chosen icon looks terribly broken when displayed
+in Zulip. There are two common issues with SVG files that must be manually
+corrected:
+
+1. [icons that rely on strokes](#correcting-icons-that-include-strokes),
+   rather than paths
+2. [icons that have an `evenodd` value](#correcting-icons-with-an-evenodd-fill-rule)
+   on the `fill-rule` attribute
+
+Each of those problems requires a different fix.
+
```

Comment:
I think it should be mentioned here that you'll need to clean up the SVG after either fix.